packages/react/src/CTABanner/CTABanner.tsx

```diff
@@ -68,6 +76,7 @@ const Root = forwardRef(
         >
           <div className={clsx(styles['CTABanner-content'], align === 'center' && styles['CTABanner-content--center'])}>
             {children}
+            {TrailingComponent && <TrailingComponent />}
```

Comment:
@joshfarrant How does this look with `align="center"`? Could you share a screenie please?

Reply:
ofc, just added it to PR description